### PR TITLE
Jossse69 patch 1

### PR DESCRIPTION
**Pull request by jossse69**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/jossse69.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "josé luiz",
+  "bio": "a newcomer-ish at open source",
```
Comment (CBID2) [inline data/jossse69.json:3]:
```suggestion
  "bio": "a newcomer-ish to open source",
```

**Review by CBID2 — APPROVED** [review]
Hi @jossse69. It looks pretty good! :) Congratulations on getting started! :)

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Comment by jossse69** [comment]
:)

**Comment by CBID2** [comment]
> :)

Yay you did it @jossse69! 😊

**Pull request merged**